src/app/InteractionModelDelegate.h

```diff
@@ -75,6 +76,38 @@ class InteractionModelDelegate
      */
     virtual CHIP_ERROR ReportError(const ReadClient * apReadClient, CHIP_ERROR aError) { return CHIP_ERROR_NOT_IMPLEMENTED; }
 
+    /**
+     * Notification that a Command Send receive Status code embeded in Invoke Command Response
+     * @param[in]  apCommandSender A current command sender which can identify the command sender to the consumer, particularly
+     * during multiple command interactions
+     * @param[in]  aGeneralCode   Status code defined by the standard
+     * @param[in]  aProtocolId    Protocol Id
+     * @param[in]  aProtocolCode  Detailed error information, protocol-specific.
+     * @param[in]  aEndpointId    Endpoint identifier
+     * @param[in]  aClusterId     Cluster identifier
+     * @param[in]  aCommandId     Command identifier
+     * @retval # CHIP_ERROR_NOT_IMPLEMENTED if not implemented
+     */
+    virtual CHIP_ERROR CommandResponseStatus(const CommandSender * apCommandSender, const uint16_t aGeneralCode,
```

Comment:
The type for aGeneralCode should probably be `chip::Protocols::SecureChannel::GeneralStatusCode`, no?

Reply:
Yes, just file another ticket for follow-up cleanup since there are too many place needs to be replaced, and don't wanna mess up this PR, thanks

Reply:
https://github.com/project-chip/connectedhomeip/issues/5547